service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -163,7 +165,7 @@ void shouldNotUpdateStateToIssueResolutionWhenReviewDecisionIsNotSendToAllPartie
     }
 
     @Test
-    void shouldNotUpdateStateToIssueResolutionWhenNextHearingIsNotOfTypeIssueResolution() {
+    void shouldNotUpdateStateToIssueResolutionOrFinalHearingWhenNextHearingIsNotOfTypeIssueResolution() {
```

Comment:
maybe rename to shouldNotUpdateStateToIssueResolutionOrFinalHearingWhenNextHearingIsNotOfEitherType ?